zerver/views/custom_profile_fields.py

```diff
@@ -78,6 +78,13 @@ def is_default_external_field(field_type: int, field_data: ProfileFieldData) ->
     return True
 
 
+# We allow updating only Name and Hint field of default external account field.
+def updating_name_and_hint_of_default_external_account(
+    existing_field_data: ProfileFieldData, requested_field_data: ProfileFieldData
+) -> bool:
+    return existing_field_data["subtype"] == requested_field_data["subtype"]
+
+
 def validate_custom_profile_field(
     name: str, hint: str, field_type: int, field_data: ProfileFieldData
 ) -> None:
```

Comment:
This function needs to be changed to no longer skip validation for name/hint on default external fields.